### ISSUE TITLE
Webcam access Error

### ISSUE THREAD

**Issue by djlolito** [kiwiirc/plugin-conference#43]
Good morning,

I wanted to indicate that the access to the microphone or the camera does not work, I have tried from Chrome and Firefox, and I get the following error:
![Captura](https://user-images.githubusercontent.com/5112090/61583287-4bcfbd00-ab36-11e9-86f2-e5c0a81de18c.PNG)

Can you help me ? Thx

**Comment by ojamin** [comment]
Hi,

That is a permissions issue in the browser, hopefully the following information (based on using Chrome) can help you resolve it.

First check you have the right mic and camera selected in the conference settings by clicking the 3 dots in the bottom right of the conference window and choosing settings
![image](https://user-images.githubusercontent.com/201050/61583411-7ec28280-ab2f-11e9-8752-7b272eebb9e3.png)

Turn on the mic and camera with the appropriate icons
![image](https://user-images.githubusercontent.com/201050/61583453-fd1f2480-ab2f-11e9-9836-1ed6ab2ad3ad.png)

Then click the icon (usually a camera icon) at the right hand side of the address bar and enable permissions for the site for your mic and camera
![image](https://user-images.githubusercontent.com/201050/61583369-18d5fb00-ab2f-11e9-8e7e-669ab7a200f3.png)

**Comment by djlolito** [comment]
I´m trying with Chrome .
When i try to enter to Settings i dont see any opcion i attach a capture.
![Captura](https://user-images.githubusercontent.com/5112090/61584736-6feac880-ab4d-11e9-9aee-225fc1ddcbc2.PNG)

**Comment by ojamin** [comment]
Try looking in site settings by clicking the lock icon at the right of the address bar to see if settings in there can enable the camera and mic.

![image](https://user-images.githubusercontent.com/201050/61585418-269d7780-ab53-11e9-8a24-cd801962ce57.png)

**Comment by djlolito** [comment]
Hello
I have manually set the permissions of camera and microphone, attached capture of how I have them. Continue with the same error.
The browser does not request camera or microphone permissions when it is accessed, I have tried it from Firefox and Chrome with the same result
I downloaded the last commit of kiwiirc, in case the error was that, but it continues.
![Captura](https://user-images.githubusercontent.com/5112090/61588287-2922c000-ab99-11e9-8c11-85415dad3018.PNG)

**Comment by ojamin** [comment]
Please can you test on Kiwiirc.com at the following link https://kiwiirc.com/nextclient/?plugins=conference

**Comment by djlolito** [comment]
Hello
From that link, the Settings option goes as in the capture, you can not see anything.
However the browser and request permission to the microphone and webcam and they work.
attached capture.

![Captura](https://user-images.githubusercontent.com/5112090/61591096-40c26e80-abc2-11e9-8905-000dcc437dda.PNG)
![Captura2](https://user-images.githubusercontent.com/5112090/61591097-40c26e80-abc2-11e9-898f-68e64ca06f3c.PNG)

**Comment by ojamin** [comment]
Can you try on https://meet.jit.si/TenseMushroomsHurryWonderfully to see if it works on pure Jitsi please.

**Comment by djlolito** [comment]
> Can you try on https://meet.jit.si/TenseMushroomsHurryWonderfully to see if it works on pure Jitsi please.

yes on this link works good. But using the last commit of kiwirc not

**Comment by ojamin** [comment]
Ok, that website is running a slightly newer version of the Jitsi Meet api - by the looks of it v3512 vs v3387 we are running on kiwiirc.com. We will look at updating to a newer version of Jitsi Meet and if you can get upgraded then hopefully that should fix your issue.

Could you tell me what OS, browser version etc your using to test so I can try and replicate and verify our upgrade when we have done it please?

**Comment by djlolito** [comment]
Hello

I'm compiling the source (kiwiirc and the plugin) on Centos 7
Server version: Apache / 2.4.6 (CentOS)
to access it try from chrome version 75.0.3770.100


If you need more information, tell me and I'll gladly give it to you

**Comment by ojamin** [comment]
Are you accessing the kiwi install from Chrome on CentOS or on Windows? If CentOS can you try connecting to that install from Chrome on Windows and confirming if the issue still persists please.

**Comment by djlolito** [comment]
> Are you accessing the kiwi install from Chrome on CentOS or on Windows? If CentOS can you try connecting to that install from Chrome on Windows and confirming if the issue still persists please.

I access the kiwiirc from Chrome and firefox using Windows.

**Comment by ojamin** [comment]
From the test above it looks like a version of jitsi-meet newer than latest stable may solve your issue. You can find them at https://github.com/jitsi/jitsi-meet
